### PR TITLE
Paint can

### PR DESCRIPTION
**Pull request by ericrius1**
Spray Paint can now resets back onto table if user drops it

sprayPaintSpawner.js spawns a sprayPaint can in correct location on table

### CODE REVIEW

File: examples/entityScripts/sprayPaintCan.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 (function() {
-    Script.include("../libraries/utils.js");
+    // Script.include("../libraries/utils.js");
+    //Need absolute path for now, for testing before PR merge and s3 cloning. Will change post-merge
+    Script.include("https://hifi-public.s3.amazonaws.com/scripts/libraries/utils.js");
```
Comment (sethalves) [inline examples/entityScripts/sprayPaintCan.js]: why not a relative url, here?

Comment (ericrius1) [inline examples/entityScripts/sprayPaintCan.js]: Using absolute paths is necessary for testing until a merge occurs- once that does, then the scripts are copied into S3, and I can change it

File: examples/entityScripts/sprayPaintCan.js [hunk 2]
```diff
@@ -25,6 +27,14 @@
         running: true
     });
 
+    var timeSinceLastMoved = 0;
+    var RESET_TIME_THRESHOLD = 5;
+    var DISTANCE_FROM_HOME_THRESHOLD = .5;
```
Comment (sethalves) [inline examples/entityScripts/sprayPaintCan.js]: zero before the decimal

https://docs.highfidelity.com/docs/coding-standard#section-3-5-2-floating-point-constants-should-always-be-written-with-decimal-point-and-at-least-one-decimal-

(I had to look it up)

File: examples/sprayPaintSpawner.js [hunk 3]
```diff
@@ -8,67 +8,33 @@
 //  Distributed under the Apache License, Version 2.0.
 //  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
 
-var scriptURL = "entityScripts/sprayPaintCan.js";
+//Just temporarily using my own bucket here so others can test the entity. Once PR is tested and merged, then the entity script will appear in its proper place in S3, and I wil switch it
+var scriptURL = "https://hifi-public.s3.amazonaws.com/eric/scripts/sprayPaintCan.js?=v1" + Math.random();
```
Comment (sethalves) [inline examples/sprayPaintSpawner.js]: relative url... the random number will break http caching, which I suppose is the point, but let's remove that bit now that testing is done.

File: examples/sprayPaintSpawner.js [hunk 4]
```diff
@@ -8,67 +8,33 @@
...
+// var scriptURL = "https://hifi-public.s3.amazonaws.com/scripts/entityScripts/sprayPaintCan.js";
+// var scriptURL = "file:////Users/ericlevin/hifi/examples/entityScripts/sprayPaintCan.js?=v1" + Math.random();
```
Comment (sethalves) [inline examples/sprayPaintSpawner.js]: left-over commented out lines

File: examples/entityScripts/sprayPaintCan.js (hunk 1 above)
Comment (sethalves) [inline examples/entityScripts/sprayPaintCan.js]: check out the top of edit.js for a nice way to handle it

**Pull request merged**